Add fixes in the poll element (#94)
bsi-software/bsi-cx-design-master-template-web: CHANGELOG.md

## Next
- Remove default elements from header-image dropzone
- Add style colors for dark mode

> - Fix poll element dark mode version
> - Fix poll element tab navigation

Reviewer: ```suggestion
- Update dark mode and accessibility for poll element
```
